CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
```

Comment:
⛏add blank line after `### Fixed`
```suggestion
### Fixed

```

Reply:
the formatting on https://keepachangelog.com/en/1.0.0/ has no spacing after titles (and in my experience, not adding spaces helps during merges)